shuup/front/views/category.py

```diff
@@ -43,7 +44,9 @@ def get_context_data(self, **kwargs):
             shop_products__shop=self.request.shop,
             variation_parent__isnull=True,
             shop_products__categories=category
-        ).filter(get_query_filters(self.request, category, data=data)).distinct()
+        ).filter(get_query_filters(self.request, category, data=data))
+        products = get_product_queryset(products, self.request, category, data)
+        products = products.distinct()
```

Comment:
I think `product = get_product_queryset(products, self.request, category, data).distinct()` might work as well. or does it? :)